examples/src/demos/Friction.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Canvas } from '@react-three/fiber'
+import type { BoxProps, PlaneProps, Triplet } from '@react-three/cannon'
+import { Physics, useMaterial, usePlane, useBox } from '@react-three/cannon'
+import { OrbitControls } from '@react-three/drei'
+
+function Box(props: BoxProps) {
+  const boxSize: Triplet = [1, 1, 1]
+  const [ref] = useBox(() => ({ args: boxSize, mass: 10, ...props }))
+  return (
+    <mesh ref={ref} castShadow receiveShadow>
+      <boxBufferGeometry args={boxSize} />
+      <meshLambertMaterial color="white" />
+    </mesh>
+  )
+}
+
+function Plane(props: PlaneProps) {
+  const [ref] = usePlane(() => ({ ...props }))
+  return (
+    <mesh ref={ref} receiveShadow>
+      <planeBufferGeometry args={[10, 5]} />
+      <meshStandardMaterial color="lightsalmon" />
+    </mesh>
+  )
+}
+
+function PhysicsContent() {
+  const groundMaterial = useMaterial(() => ({ friction: 0.5 }))
```

Comment:
How is `useMaterial` better than passing materialOptions to one of the physics body hooks?